T2_US_Caltech/FE/main.yaml

```diff
@@ -0,0 +1,61 @@
+  sites: 
+    - "T2_US_Caltech"
+  webdomain: "https://sense-caltech-fe.tier2-lb.ultralight.org:443"
+  probes: ['https_v4_siterm_2xx', 'icmp_v4']
+T2_US_Caltech: 
+  domain: ultralight.org
+  latitude: 34.1377
+  longitude: 118.1253
+  plugin: ansible
+  privatedir: /opt/siterm/config/T2_US_Caltech/
+  ipv6-subnet-pool: [2605:d9c0:6:2640::/64,2605:d9c0:6:2641::/64,2605:d9c0:6:2642::/64,2605:d9c0:6:2643::/64,2605:d9c0:6:2644::/64,2605:d9c0:6:2645::/64,2605:d9c0:6:2646::/64,2605:d9c0:6:2647::/64,2605:d9c0:6:2648::/64,2605:d9c0:6:2649::/64,2605:d9c0:6:2650::/64,2605:d9c0:6:2651::/64,2605:d9c0:6:2652::/64,2605:d9c0:6:2653::/64,2605:d9c0:6:2655::/64]
+  metadata:
+    xrootd:
+      "2605:d9c0:6:2640::/64": "sense-redir-01.ultralight.org:1094"
+      "2605:d9c0:6:2641::/64": "sense-redir-02.ultralight.org:1094"
+      "2605:d9c0:6:2642::/64": "sense-redir-03.ultralight.org:1094"
+      "2605:d9c0:6:2643::/64": "sense-redir-04.ultralight.org:1094"
+      "2605:d9c0:6:2644::/64": "sense-redir-05.ultralight.org:1094"
+      "2605:d9c0:6:2645::/64": "sense-redir-06.ultralight.org:1094"
+      "2605:d9c0:6:2646::/64": "sense-redir-07.ultralight.org:1094"
+      "2605:d9c0:6:2647::/64": "sense-redir-08.ultralight.org:1094"
+      "2605:d9c0:6:2648::/64": "sense-redir-09.ultralight.org:1094"
+      "2605:d9c0:6:2649::/64": "sense-redir-10.ultralight.org:1094"
+      "2605:d9c0:6:2650::/64": "sense-redir-11.ultralight.org:1094"
+      "2605:d9c0:6:2651::/64": "sense-redir-12.ultralight.org:1094"
+      "2605:d9c0:6:2652::/64": "sense-redir-13.ultralight.org:1094"
+      "2605:d9c0:6:2653::/64": "sense-redir-14.ultralight.org:1094"
+      "2605:d9c0:6:2654::/64": "sense-redir-15.ultralight.org:1094"
+      "2605:d9c0:6:2655::/64": "sense-redir-16.ultralight.org:1094"
+  ipv6-address-pool: [fc00:3600::0/64,fc00:3601::0/64,fc00:3602::0/64,fc00:3603::0/64,fc00:3604::0/64,fc00:3605::0/64,fc00:3606::0/64,fc00:3607::0/64,fc00:3608::0/64,fc00:3609::0/64,fc00:3610::0/64,fc00:3611::0/64,fc00:3612::0/64,fc00:3613::0/64,fc00:3614::0/64,fc00:3615::0/64,fc00:3616::0/64,fc00:3617::0/64,fc00:3618::0/64,fc00:3619::0/64,fc00:1779::0/64,fc00:1780::0/64,fc00:1781::0/64,fc00:1782::0/64,fc00:1783::0/64,fc00:1784::0/64,fc00:1785::0/64,fc00:1786::0/64,fc00:1787::0/64,fc00:1788::0/64,fc00:1789::0/64,fc00:1790::0/64,fc00:1791::0/64,fc00:1792::0/64,fc00:1793::0/64,fc00:1794::0/64,fc00:1795::0/64,fc00:1796::0/64,fc00:1797::0/64,fc00:1798::0/64,fc00:1799::0/64,fc00:3985::0/64,fc00:3986::0/64,fc00:3987::0/64,fc00:3988::0/64,fc00:3989::0/64]
+  ipv4-address-pool: [10.251.85.0/24,10.251.86.0/24,10.251.87.0/24,10.251.88.0/24,10.251.89.0/24,172.16.3.0/30,172.17.3.0/30,172.18.3.0/30,172.19.3.0/30,172.31.10.0/24,172.31.11.0/24,172.31.12.0/24,172.31.13.0/24,172.31.14.0/24,172.31.15.0/24]
+  year: 2024
+  switch:
+    - "Edgecore-Sonic"
+Edgecore-Sonic:
+  vsw: "Edgecore-Sonic"
+  rst: "Edgecore-Sonic"
+  rsts_enabled: "ipv4,ipv6"
+  private_asn: 64516
+  vrf: lhcone
+  vlan_mtu: 9416
```

Comment:
9416 supported by sonic? (havent checked - but worth to ensure what is the max allowed on port)